src/lib/operators/join_sort_merge.cpp

```diff
@@ -122,12 +127,12 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractJoinOperatorImpl {
   struct TableRange;
   struct TablePosition {
     TablePosition() = default;
-    TablePosition(size_t cluster, size_t index) : cluster{cluster}, index{index} {}
+    TablePosition(const size_t cluster, const size_t index) : cluster{cluster}, index{index} {}
```

Comment:
Do you have a benchmark of just the constness? That would be interesting for me, but also a good motivator for the seminar.

Reply:
Oh ... that might be something I'll test at the very end.